assets/css/less/wpbdp/abstracts/variables/colors.less

```diff
@@ -0,0 +1,40 @@
+/* Color */
+@color-primary: #569af6;
```

Comment:
Do we need to add the rest of the colors from the old colors.less and delete that file?